### ISSUE TITLE
New Term Request: dean role

### ISSUE THREAD

**Issue by Ant024** [hectorguzor/provisional_OntoEd#7]
We want to create a textual definition and annotation for "dean."

**Comment by hectorguzor** [comment]
hector will work on this

**Comment by hectorguzor** [comment]
Discussed on 03/07/2024: we looked at wikipedia def for "Dean"
Dean is a title employed in [academic administrations](https://en.wikipedia.org/wiki/Academic_administration) such as colleges or universities for a person with significant authority over a specific [academic](https://en.wikipedia.org/wiki/Academia) unit, over a specific area of concern, or both. In the [United States](https://en.wikipedia.org/wiki/United_States) and [Canada](https://en.wikipedia.org/wiki/Canada), deans are usually university professors who serve as the heads of a university's constituent colleges and schools. Deans are common in private [preparatory schools](https://en.wikipedia.org/wiki/University-preparatory_school), and occasionally found in [middle schools](https://en.wikipedia.org/wiki/Middle_school) and [high schools](https://en.wikipedia.org/wiki/High_school) as well.

This class should fall under BFO "role". the group agrees that this needs improvment because it is too long of a definition. So the label should be 'dean role"
Anthony will work on this with the help of the group.

**Comment by hectorguzor** [comment]
Hector will add "dean role" to `terminology.tsv`

**Comment by Ant024** [comment]
The Dean Role is for academic leaders who have academic, programmatic, managerial, and fiscal responsibilities for a school. Deans verify the adequacy of instruction, monitor academic integrity, confer degrees, and are responsible for student recruitment, admission, and academic progress.

I seen this definition and thought it was a bit better. I will wait on comments.

**Comment by Ant024** [comment]
We discussed the need for two separate definitions for Dean. One as a role and one as an individual who satisfies that role. I will define each below.

Dean- A continuant that satisfies the requirements for Dean Role.

Dean Role- A role in educational institutions that is satisfied by having academic, programmatic, managerial and fiscal responsibilities.

**Comment by hectorguzor** [comment]
Discussed on2024-07-18: We will create a new issue for the class "dean". Anthony will work on the class 'dean role".

**Comment by hectorguzor** [comment]
'dean role' class has been created. this issue will be closed.
